usr/share/66/service/chronyd

```diff
@@ -1,15 +1,16 @@
 [main]
 @type = classic
 @description = "chrony ntp daemon"
-@version = @VERSION@ 
+@version = @VERSION@
 @user = ( root )
-@options = ( log )
+@options = ( log env )
 
 [start]
-@build = custom
-@shebang = "/bin/sh"
-@execute = ( install -d -m750 -o chrony -g chrony /var/run/chrony
-exec chronyd -n -u chrony 2>&1 )
+@execute = ( foreground { install -d /var/run/chrony  -m 750 -o chrony -g chrony }
```

Comment:
this can be made with execl-toc
`execl-toc -d /var/run/chrony -m0750 -u chrony -g chrony` as a kickstart..

Reply:
Does that override the dir always like install? Ι will like to have the exact same behavior...

Reply:
Install replaced, thanx :)